engine/gamesys/src/dmsdk/gamesys/script.h

```diff
@@ -0,0 +1,168 @@
+// Copyright 2020 The Defold Foundation
```

Comment:
Should we update this stub now?

Reply:
Ah nevermind if this file was moved

Reply:
Yes, we probably should do a separate PR with a license update.